types/express/v4/package.json

```diff
@@ -0,0 +1,36 @@
+{
+    "private": true,
+    "name": "@types/express",
+    "version": "4.17.9999",
+    "projects": [
+        "http://expressjs.com"
+    ],
+    "dependencies": {
+        "@types/body-parser": "*",
+        "@types/express-serve-static-core": "^4.17.33",
```

Comment:
You can just write `"^4"` here; doesn't need to be this specific

Reply:
```suggestion
        "@types/express-serve-static-core": "^4",
```

Reply:
Agh, sorry, I thought this was put here by you, but it's just the way the package was before. can you revert the change? Sorry :(

Reply:
yes...

Reply:
reverted